### ISSUE TITLE
Bootstrap header with button

### ISSUE THREAD

**Issue by lpugin** [DDMAL/vida.js#25]
Shouldn't we have a navbar for the navigation arrows and toggle button as in the VerovioEditor?

**Comment by ahwitz** [comment]
You mean like the full black bar on http://cantus.simssa.ca/editor/?ch-sgs-390 ?

**Comment by lpugin** [comment]
Exactly. And along the same lines, it would be more intuitive to have a refresh button in the Verovio header side that would load the mei file selected in the tabs (instead of having to select "update Verovio" and select the file). Does this make sense?

**Comment by ahwitz** [comment]
Yup to both.

**Comment by lpugin** [comment]
Where does this have to be changed? In vida.js? Sorry, I don't have a good overview of this.

**Comment by ahwitz** [comment]
Ah, I didn't realize this was an urgent thing. vida will need Bootstrap as a dependency and the necessary design changes, but that's trivial.

To get the "refresh" button... Since it relies on data in the VerovioEditor and since we had intended this to be a standalone project (if you or @ahankinson want it to be combined into the VerovioEditor repo, this could be made easier), what I'd recommend you do is along the lines of...

    function refreshButtonHandler(){this.refreshVerovio();}
    $("#refresh-button").on('click', refreshButtonHandler);
    this.updateRefreshButton = function(newFunction){refreshButtonHandler = newFunction;}

...and then call updateRefreshButton with a reference to a function that gets the text of the active page in meix.

Meix sense? (http://instantrimshot.com/) Any better ideas?

**Comment by lpugin** [comment]
Not that urgent. Most important is the organization of the code. What do you mean by having it combined in the VerovioEditor repo? vida.js?

**Comment by ahwitz** [comment]
Yeah, make vida.js solely part of the VerovioEditor repo, as I don't know if it'll be used anywhere else. Doing that will allow us to combine the vida functionality into a meix plugin and it might be easier to work with.

**Comment by lpugin** [comment]
Sounds like a high priority to me. I'll see with @ahankinson
